### PR TITLE
Clean up migrate DB calls

### PR DESCRIPTION
**Pull request by chrismwendt**
Cleans up what appears to be dead code.

### CODE REVIEW

File: internal/db/dbutil/dbutil.go [hunk 1]
```diff
@@ -153,6 +160,15 @@ func MigrateDB(db *sql.DB, dataSource string) error {
 	return err
 }
 
+type stdoutLogger struct{}
+
+func (logger stdoutLogger) Printf(format string, v ...interface{}) {
```
Comment (creachadair) [inline internal/db/dbutil/dbutil.go]: Suggestion: When a method doesn't use its receiver, the name of the receiver may be omitted
```suggestion
func (stdoutLogger) Printf(format string, v ...interface{}) {
```

This is a helpful signal for the reader too.

Even when it's not omitted, a [shorter receiver name](https://github.com/golang/go/wiki/CodeReviewComments#receiver-names) is probably preferable.

Comment (chrismwendt) [reply internal/db/dbutil/dbutil.go]: That is a helpful signal, I like that.

File: cmd/frontend/db/dbstore_db_test.go [hunk 2]
```diff
@@ -16,12 +17,15 @@ func TestMigrations(t *testing.T) {
 	// Setup a global test database
 	dbtesting.SetupGlobalTestDB(t)
 
-	m := dbconn.NewMigrate(dbconn.Global, "")
+	m, err := dbutil.NewMigrate(dbconn.Global, "")
+	if err != nil {
+		t.Errorf("error constructing migration migrations: %s", err)
```
Comment (unknwon) [inline cmd/frontend/db/dbstore_db_test.go]: typo?
```suggestion
		t.Errorf("error constructing migrations: %s", err)
```

File: internal/db/dbtest/dbtest.go [hunk 3]
```diff
@@ -68,7 +68,11 @@ func NewDB(t testing.TB, dsn string) (*sql.DB, func()) {
 	config.Path = "/" + dbname
 	testDB := dbConn(t, config)
 
-	if err = dbutil.MigrateDB(testDB, dsn); err != nil {
+	m, err := dbutil.NewMigrate(testDB, dsn)
+	if err != nil {
+		t.Fatalf("failed to construct migration: %s", err)
```
Comment (unknwon) [inline internal/db/dbtest/dbtest.go]: I saw every other places use plural:
```suggestion
		t.Fatalf("failed to construct migrations: %s", err)
```

File: internal/db/dbutil/dbutil.go [hunk 4]
```diff
@@ -14,6 +14,7 @@ import (
 	// Register driver
 	_ "github.com/lib/pq"
 
+	"github.com/golang-migrate/migrate/v4"
 	migr "github.com/golang-migrate/migrate/v4"
```
Comment (unknwon) [inline internal/db/dbutil/dbutil.go]: It looks like these two are the same package?

Comment (chrismwendt) [reply internal/db/dbutil/dbutil.go]: Yup, cleaned up imports, thanks.

File: internal/db/dbutil/dbutil.go [hunk 5]
```diff
@@ -153,6 +163,15 @@ func MigrateDB(db *sql.DB, dataSource string) error {
 	return err
 }
 
+type stdoutLogger struct{}
+
+func (stdoutLogger) Printf(format string, v ...interface{}) {
+	fmt.Printf(format, v...)
+}
+func (logger stdoutLogger) Verbose() bool {
```
Comment (creachadair) [inline internal/db/dbutil/dbutil.go]:
```suggestion
func (stdoutLogger) Verbose() bool {
```

It's also stylistically fine to make short functions single line, if you want to save some vertical space.

**Pull request merged**